packages/containerd/containerd-cri-base-json

```diff
@@ -102,6 +102,15 @@ oci-defaults = { version = "v1", helpers = ["oci_defaults"] }
                 "mode=755",
                 "size=65536k"
             ]
+        },
+        {
+            "destination": "/usr/local/sbin/modprobe",
```

Comment:
Have we confirmed this will be used instead of the provided `modprobe` for containers that include it? Do we know where this path related in the default PATH for these containers?

Reply:
The [linked script] in the description printed the default `PATH` in the container images. For all of them `/usr/local/sbin` is the first path in the list.

[linked script]: https://gist.github.com/bcressey/e6a4e30ef2f0d886a09c19ecfbdc7d1c